frontends/web/src/routes/account/send/utxos.tsx

```diff
@@ -76,64 +77,77 @@ export class UTXOsClass extends Component<Props, State> {
         });
     }
 
-    public render() {
-        const { t, active, explorerURL, onClose } = this.props;
+    private renderUTXOs = (scriptType: accountApi.ScriptType) => {
+        const { t, explorerURL } = this.props;
         const { utxos, selectedUTXOs } = this.state;
+        const filteredUTXOs = utxos.filter(utxo => utxo.scriptType === scriptType);
+        if (filteredUTXOs.length === 0) {
+            return null;
+        }
+        return (
+            <div>
+            <h1>{ getScriptName(scriptType) }</h1>
```

Comment:
@thisconnect is `h1` appropriate? or better another tag/style?

Reply:
`<h2 className="subTitle">` is mostly used for  titles other than the maintitle.

Reply:
thx, done